gui.js

```diff
@@ -3041,40 +3040,144 @@ IDE_Morph.prototype.popupMediaImportDialog = function (folderName, items) {
     // private - this gets called by importMedia() and creates
     // the actual dialog
     var dialog = new DialogBoxMorph().withKey('import' + folderName),
+        content = new Morph(),
         frame = new ScrollFrameMorph(),
         selectedIcon = null,
         turtle = new SymbolMorph('turtle', 60),
         myself = this,
         world = this.world(),
-        handle;
+        handle,
+        msg,
+        count,
+        listFieldWidth = 180,
+        isSound = (folderName.toLowerCase() === "sounds"),
+        buttonColor = new Color(237, 237, 237);
+    buttonHighlightColor = buttonColor.darker();
+    buttonPressColor = buttonHighlightColor.darker().darker();
+    buttonTextColor = new Color(100, 100, 100);
+
+    var itemsSortFilter = function (a, b) {
+        if (a.name.toLowerCase() < b.name.toLowerCase()) return -1;
+        else return 1;
+    };
+
+    var sectionsSortFilter = function (a, b) {
+        if (a.name === "other") return 1;
+        if (b.name === "other") return -1;
+        return itemsSortFilter(a, b);
+    };
+
+    var colect = {};
+    items.forEach(function (item) {
+        var section = (item.section === "") ? "other" : item.section;
```

Comment:
I think we might to make the check more broad... Maybe `isNil(item.section) || item.section === ''` ?

Otherwise if someone leaves off a section key then the sections list would have a "null" section.